core/modules/locale/views/locale.views.inc

```diff
@@ -218,4 +218,23 @@ function locale_views_data_alter(&$data) {
       'handler' => 'views_handler_sort',
     ),
   );
+
+  // Term language field
```

Comment:
Same as my previous comment, inline comments need to be full sentences, and must have a period at the end.